### ISSUE TITLE
We need a name.

### ISSUE THREAD

**Issue by MitchWeaver** [MitchWeaver/bonsai#7]
`distro` is a placeholder.

Names are hard.

Give me your suggestions

**Comment by Jackojc** [comment]
Bushmaster Linux

**Comment by MitchWeaver** [comment]
`bonsai` has been suggested and I'm quite liking it --  @zoefiri even made this *adorable* ascii logo for us

```
  ####     
#######,  ######
  ###',#  #######
   ###'';,,,'###
       ,;  ''''
      ;;;   ,#####,
     ;;;'  ,,;;;###
     ';;;;'''####'
      ;;;     
   ,.;;';'',,,
  '     '
 #                                  
 #                                  
 ##, ,##,',##, ,##  ,#,   o         
 # # #  # #''# #,,  # #   #         
 '#' '##' #  #  ,,# '##;, #   
```

What does everyone think?

**Comment by thotslayer** [comment]
I like it, it's pretty cute

**Comment by Jackojc** [comment]
Bonsai sounds like it's for weak motherfuckers. This distro is for hard motherfuckers. You gotta be a bushmaster, a man of the land, a fucking survivor to use it.

**Comment by ZoeFiri** [comment]
> Bonsai sounds like it's for weak motherfuckers. This distro is for hard motherfuckers. You gotta be a bushmaster, a man of the land, a fucking survivor to use it.

bruh moment

**Comment by MitchWeaver** [comment]
name has been decided. we are now `bonsai`.

closed by cfb279e9c7e1be0d73e5b250ad0ce5373852895f
